app/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,6 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Consider replacing it with READ_EXTERNAL_STORAGE one

Reply:
Does the WRITE_ include the READ_ permission...? I thought they were different permission, and they are basically does not required since api level 19. This apk now has minSdkVersion 21, so... perhaps we can just remote this.

READ_EXTERNAL_STORAGE
> Also starting in API level 19, this permission is not required to read or write files in your application-specific directories returned by [Context.getExternalFilesDir(String)](https://developer.android.com/reference/android/content/Context#getExternalFilesDir(java.lang.String)) and [Context.getExternalCacheDir()](https://developer.android.com/reference/android/content/Context#getExternalCacheDir()).

WRITE_EXTERNAL_STORAGE
> Starting in API level 19, this permission is not required to read/write files in your application-specific directories returned by [Context.getExternalFilesDir(String)](https://developer.android.com/reference/android/content/Context#getExternalFilesDir(java.lang.String)) and [Context.getExternalCacheDir()](https://developer.android.com/reference/android/content/Context#getExternalCacheDir()).